### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by cavemansspa** [ipavlic/apex-fp#21]
nice work.

is there a way to install (managed package?) so that the apex-fp tests are not part of the organization test  runs? thinking i may have installed incorrectly as the apex-fp test runs are running with our code deploys.

**Comment by ipavlic** [comment]
No, I don't provide that. It's harder to rename and deprecate things and I would have to maintain essentially two code bases, with and without the namespace.

You've "installed" it correctly — it's just code that you have deployed to your org. But instead of you having to come up with ideas and write it, you can use just it because it's open source. You can also report bugs, or suggest your improvements. Or you can change the direction of where the library is going and make it your own by forking it.

I'm sharing my work with you for free. I am also paying for a domain name out of my pocket so that there's nicer searchable documentation available.
